source/appModules/devenv.py

```diff
@@ -102,11 +110,30 @@ def chooseNVDAObjectOverlayClasses(self, obj, clsList):
 				pass
 			clsList.insert(0, VsTextEditPane)
 
-		if ((vsMajor == 15 and vsMinor >= 3)
-			or vsMajor >= 16):
+		if (
+			(self.major == 15 and self.minor >= 3)
+			or self.major >= 16
+		):
 			if obj.role == controlTypes.ROLE_TREEVIEWITEM and obj.windowClassName == "LiteTreeView32":
 				clsList.insert(0, ObjectsTreeItem)
 
+			if (
+				obj.role == controlTypes.ROLE_MENUITEM
+				and isinstance(obj, UIA.UIA)
+				and obj.UIAElement.CachedClassName in ("IntellisenseMenuItem",)
+			):
+				clsList.insert(0, IntellisenseMenuItem)
+
+			try:
+				if (
+					obj.role == controlTypes.ROLE_STATICTEXT

```

Comment:
I"m afraid that this code will have major impact on performance, as it will create several UIA objects for objects we do not use.

Reply:
How can I avoid this issue?

Thanks,